template/.claude/commands/summarize.md

```
@@ -0,0 +1,62 @@
+---
+description: Summarize conversation progress and next steps
+argument-hint: [optional additional info]
+---
+
+## General Guidelines
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)